README.md

```diff
@@ -918,6 +918,23 @@ Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oi
       authScheme: 'OAUTH2'
     }
     ```
+- **customButtons** - Adds an array of custom Auth buttons configurable by the properties below:
+
+    - `title` - String to set button text
```

Comment:
We *mostly* don't word things like this. Actually, this is a good reminder, @mystiberry-okta can you check the text in this PR? This is going to show up in our widget README (and later on, on d.o.c.).

Better perhaps to say something like "the button text"